docs/docfiles/user_guides/QuickStart.rst

```diff
@@ -36,12 +36,9 @@ On Windows
 
 .. note::
 
-    The ``dpctl.lsplatform()`` function is new in dpctl 0.7 and will be
-    available in oneAPI 2021.3. If you are following the guide on an older
-    oneAPI installation, use ``dpctl.dump()``. If no GPU platforms are shown,
-    make sure your system has a supported GPU and has the necessary GPU drivers
-    installed. You can install GPU drivers by following the
-    `GPU driver installation guide`_.
+    If no GPU platforms are shown, make sure your system has a supported
+    GPU and the necessary GPU drivers installed. See `GPU driver installation guide`_ to install GPU drivers.
+    drivers by following the `GPU driver installation guide`_.
```

Comment:
This last line is incomplete and a repetition. Please remove.

Reply:
Done.